app/views/projects/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :pre_content_nav do %>
+  <% render partial: "shared/back_link", locals: {href: root_path} %>
+<% end %>
+
+<% content_for :page_title do %>
+  <%= page_title(t("new_project_form.title")) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @new_project_form, url: projects_path, method: :post do |form| %>
+      <%= form.govuk_error_summary %>
+
+      <%= form.govuk_collection_radio_buttons :project_type,
+            @new_project_form.project_types,
+            ->(option) { option },
```

Comment:
Fancy

Reply:
Straight out of the docs: 
https://govuk-form-builder.netlify.app/form-elements/radios/#populating-values-labels-and-hints-with-procs

👍